src/components/CalendarContainer.tsx

```diff
@@ -248,8 +254,11 @@ function _CalendarContainer<T extends ICalendarEventBase>({
         }
       }
       setTargetDate(nextTargetDate)
+      if (onSwipe) {
+        onSwipe(nextTargetDate.toDate())
+      }
     },
-    [swipeEnabled, targetDate, mode, theme.isRTL],
+    [swipeEnabled, theme.isRTL, onSwipe, targetDate, mode],
```

Comment:
It sounds like it's added to the dependencies list of the memorized `onSwipeHorizontal` callback by the internal implementation.
Should we add a note to the documentation that `onSwipe` (or `onSwipEnd`) should memorized to avoid unnecessary re-renders?

Reply:
I added a note